flannel/flannel-config.md

```diff
@@ -147,6 +147,10 @@ The last step is to enable `flanneld.service` in the Ignition config:
 
 *Important*: Other units that will run in containers, including those scheduled via fleet, should include `Requires=flanneld.service`, `After=flanneld.service`, and `Restart=always|on-failure` directives. These directive are necessary because flanneld.service may fail due to etcd not being available yet. It will keep restarting and it is important for Docker based services to also keep trying until flannel is up.
 
+### Setting the environment variables
+
+If etcd requires SSL certificates, flannel will need those to communicate with etcd. This is achieved by passing the directory in which the certificates are stored to the flannel wrapper. To do so, set the environment variable, `ETCD_SSL_DIR` to `/etc/ssl` in `/etc/flannel/options.env`. 
```

Comment:
I take it back, we shouldn't read from `/etc/flannel/options.env`. The only reason `/run/flannel/options.env` exists is for legacy reasons (working around docker).